_episodes/12-cluster.md

```diff
@@ -23,143 +27,296 @@ On the first day, her supervisor greets her friendly and welcomes her to the job
 
 Lola walks to the IT department and finishes the paper work to get an account. One of the admins promises to sit down with her in the morning to show her the way around the machine. The admin explains that Lola will use a small to mid-range HPC cluster.
 
-{% include links.md %}
+## Going remote
+
+First of all, the admin asks Lola to connect to the super computer. She mentions that in the past, compute clusters were named after planets or moons as they often presented distant somewhat mythological places. One of the admin's first instructors then often said, that they would use the Space Shuttle (or `ssh` briefly) to reach that planet or moon. So the admin asks Lola to open a terminal on her laptop and type in the following commands:
```

Comment:
This implies that `ssh` stands for "**S**pace **Sh**uttle," rather than "**s**ecure **sh**ell". This can only lead to confusion. I would prefer to avoid strained metaphors such as this, unless @psteinb and other instructors have evidence of it working as a mnemonic device?

Reply:
good point, I'll rephrase.

Reply:
I dropped the passage about the space shuttle now. See f7cd03f